watchtower/wtserver/server.go

```diff
@@ -86,11 +88,31 @@ func New(cfg *Config) (*Server, error) {
 		cfg.ChainHash,
 	)
 
+	// Now that we've established that this session does not exist in the
```

Comment:
yes this is a temporary meausre until the server db has been merged, limiting it generating one addresser per restart when using an in-memory database. the server has no way to correlate sessions/updates to a particular channel, so it can't necessarily do this in the same way the client can. the final approach will likely be to keep a running count of the number of state updates allocated to a partiular address, and rotate to a new reward address after every ~N. I'm thinking this would occur on the order of 100k ~ 1M, though really depends on the relative frequency of breaches/updates-sent-to-this-tower. With some real data in the wild, it should be easier to tune this value (the tower could even self adjust in theory)